packages/utils/src/internals/cheerio.ts

```diff
@@ -0,0 +1,86 @@
+import cheerio, { load } from 'cheerio';
```

Comment:
i would move this to the cheerio package, wdyt? given how small the crawler packages are, we can just export those directly, or have a namespace export like `export * as utils`

Reply:
Doesn't really make sense, as its not a cheerio crawler specific util... I guess its iffy

Reply:
how come its not cheerio specific when it uses cheerio? i see it is used in some other utils, but those also look cheerio specific and should be moved to the cheerio package i'd say

Reply:
Well, you can pass in any strings to it (so you could technically use a puppeteer page, get the html and pass it to this util)... but yeah I can see the confusion it can cause.. its your call really

Reply:
will get some feedback from Ondra, for me the goal is to not depend on cheerio anywhere except the cheerio crawler (or have it as optional peer dependency, but that feels more like a hack for this case)

(definitely not a blocker for merging this PR)